lightning/src/ln/features.rs

```diff
@@ -231,6 +233,16 @@ mod sealed {
 				}
 			}
 
+			impl <T: $feature> Features<T> {
+				pub fn $supports_setter(&mut self) {
```

Comment:
I think we want to avoid overloading the word "supports" which is used elsewhere to mean has either the optional or required bit set whereas you are using it here to only set the optional bit.